README.md

```diff
@@ -1 +1,131 @@
-# Hello-Microverse
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="images/coder.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is created by HTML and CSS to be showed a green colored Hello Microverse Title.
+
+
+## 🛠 Built With <a name="built-with"></a>
+Html and CSS
+
+
+
+### Key Features <a name="key-features"></a>
+
+- **[CSS file is externally linked to HTML]**
+- **[h1 tag is colored to green]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Code editor(VS), Web browser and Git.
+
+
+### Setup
+
+Clone this repository to your desired folder: https://github.com/kakalygaday/Hello-Microverse
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command: 
+Open the file with your default brower or use live server extension
+
+Example command:
+
+```sh
+  rails server
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Abdy**
+
+- GitHub: [@githubhandle](https://github.com/kakalygaday)
+
+
+- [ ] **[Change the color of h1 tag]**
```

Comment:
- [ ] You have done extremely well in implementing the change requested by the previous reviewer, However, while making those changes you may have mistakenly removed the `Future-Feature` Section, Please add that and remember, it should be a minimum of two future features not one as I can see you have only one that says: `[Change the color of h1 tag]` By documenting future features in the README file, developers can have a centralized location for tracking planned enhancements. It serves as a reminder for the development team and provides a reference for future updates. 🙂